test/anvil_test.exs

```diff
@@ -1,7 +1,131 @@
 defmodule AnvilTest do
   use ExUnit.Case
 
-  test "the truth" do
-    assert 1 + 1 == 2
+  defmodule TestRepo do
+    def insert!(record) do
+      send self, {:created, record}
+      record
+    end
+  end
+
+
+  defmodule MyApp.Anvil do
+    use Anvil
+
+    def fixture(:without_opts) do
```

Comment:
I feel like this function is a fixture. The other functions are factory functions to do something with the fixture. Maybe that's wrong though :S